build.sbt

```diff
@@ -231,11 +231,8 @@ lazy val publishSettings = Seq(
       "scm:git:git@github.com:laserdisc-io/laserdisc.git"
     )
   ),
-  homepage := Some(url("http://laserdisc.io")),
   licenses := Seq("MIT" -> url("https://raw.githubusercontent.com/laserdisc-io/laserdisc/master/LICENSE")),
-  pgpPublicRing := file(".travis/local.pubring.asc"),
-  pgpSecretRing := file(".travis/local.secring.asc"),
-  releaseEarlyWith := SonatypePublisher
+  pgpPublicRing := file(".travis/local.pubring.asc")
```

Comment:
I don’t think we need this either, do we?

Reply:
ops :)